routing/chainview/bitcoind.go

```diff
@@ -479,3 +486,10 @@ func (b *BitcoindFilteredChainView) FilteredBlocks() <-chan *FilteredBlock {
 func (b *BitcoindFilteredChainView) DisconnectedBlocks() <-chan *FilteredBlock {
 	return b.blockQueue.staleBlocks
 }
+
+// GetBlock is used to retrieve the block with the given hash. This function
+// wraps the blockCache's GetBlock function.
+func (b *BitcoindFilteredChainView) GetBlock(hash *chainhash.Hash) (
+	*wire.MsgBlock, error) {
```

Comment:
nit: newline here since we wrap the function, last example in [code spacing guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#code-spacing).